source/Playnite/Database/GameDatabase.cs

```diff
@@ -1310,6 +1313,37 @@ bool updateCompletionStatus(Game game, CompletionStatusSettings settings)
             }
         }
 
+        public List<FilterPreset> GetSortedFilterPresets(bool getClone)
+        {
+            var filterPresetsSettings = (FilterPresets as FilterPresetsCollection).GetSettings();
+            var sortingOrder = filterPresetsSettings.SortingOrder;
+            var filterPresets = getClone ? FilterPresets.GetClone() : FilterPresets;
+            return filterPresets.OrderBy(x =>

```

Comment:
This will not sort items properly. Items that are not on the sort list settings should be sorted alphabetically at the end of the list. You are just adding them to then end of the list in order in which they are in the library.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3258/commits/b0495d4d016a7f671adb5ce28b1e17c9ac43fd42